apps/cms/pages/models.py

```diff
@@ -15,31 +19,80 @@ class HomePage(Page):
         verbose_name="Header Image",
         help_text="The Image that is shown on top of the page"
     )
-    subtitle = models.CharField(
+
+    subtitle_de = models.CharField(
+        max_length=500, blank=True, verbose_name="Subtitle")
+    subtitle_en = models.CharField(
         max_length=500, blank=True, verbose_name="Subtitle")
-    body = RichTextField(blank=True)
 
-    subpage_types = ['a4_candy_cms_pages.EmptyPage']
+    body_de = RichTextField(blank=True)
+    body_en = RichTextField(blank=True)
+
+    body = TranslatedField(
+        'body_de',
+        'body_en'
+    )
+
+    subtitle = TranslatedField(
```

Comment:
nitpick: the order is a little bit confusing (first subtitle, then body, then bode translated field, then subtitle translated field)

Reply:
As we're gonna replace this again, soon, lets just merge it :)